### PR TITLE
update ces alarm rule test

### PR DESCRIPTION
**Pull request by zengchen1024**

### CODE REVIEW

**Comment by gator1** [comment]
we have a problem with gophercloud. Right now for terraform_opentelekom_cloud I used my gophercloud from gator1/gophercloud and add our features such as elb, I then use govendor to update the gophercloud under vendor of opentelekom. Because vendor directory doesn't really hold git history, this gives a history in what we changed in gophercloud. Last time due to the time
restrict I didn't enforce this. Now if all of us just change gophercloud under vendor we will not only lose git history but also the ability to use govendor since we don't have a common repository. 
So I require that you test your changes with gophercloud against my gator1/gophercloud .

The open source community already raised the issue that we have a gophercloud under vendor but it's not a gophercloud, we modified the original gophercloud and have our own, but we still call it gophercloud. I am in the process of figuring out if we can use the standard gophercloud and add our modification to something like huaweicloud. 

In summary, for now please first do a pull request for gophercloud under gator1/gophercloud.

**Comment by gator1** [comment]
I tried to split our (modified) gophercloud into the original gophercloud and huaweicloud, huaweicloud has all our stuff such as ces, elb. 

I modified gator1/huaweicloud to include all the new stuff.

in gator1/terraform-provider-opentelekomcloud I tried to make the references to the (new) right places. I did these under a new branch, huaweicloud. 

I got most of the stuff built. Now I have this

MacBook-Pro:terraform-provider-opentelekomcloud G00725902$ make build
==> Checking that code complies with gofmt requirements...
go install
# github.com/gator1/huaweicloud/openstack/cloudeyeservice/alarmrule
../huaweicloud/openstack/cloudeyeservice/alarmrule/urls.go:10:23: c.ProjectID undefined (type *gophercloud.ServiceClient has no field or method ProjectID)
../huaweicloud/openstack/cloudeyeservice/alarmrule/urls.go:14:23: c.ProjectID undefined (type *gophercloud.ServiceClient has no field or method ProjectID)
../huaweicloud/openstack/cloudeyeservice/alarmrule/urls.go:18:23: c.ProjectID undefined (type *gophercloud.ServiceClient has no field or method ProjectID)
# github.com/gator1/huaweicloud/openstack/networking/v2/extensions/elbaas/loadbalancer_elbs
../huaweicloud/openstack/networking/v2/extensions/elbaas/loadbalancer_elbs/requests.go:197:14: c.Delete2 undefined (type *gophercloud.ServiceClient has no field or method Delete2)
make: *** [build] Error 2

I have no idea why this cloudeye stuff refers to ProjectID. Why do you need to change gophercloud's project_client.go file and add a ProjectID there? Could you remove it? This is causing issues to our future work. We can't modify gophercloud, the owner protested. I can't see any history. The vendor directory is controlled by govendor and you are not supposed to modify it directly.

**Comment by freesky-edward** [comment]
I totally agree to use huaweicloud sdk instead of gophercloud or any other repository. as far as I know huawei's SDKs are all located in [here](https://github.com/huaweicloudsdk), there already have many guys starting working on this. please consider contributing all the changes to there.

**Comment by gator1** [comment]
@freesky-edward I prefer (the open source community as well) using standard gophercloud and then have an OTC specific sdk. So whatever gophercloud already has, we use gophercloud. 
I don't know what project is using the Huawei SDK. To use it for TerraForm sounds like it needs a lot of test at least.

**Comment by zengchen1024** [comment]
@gator1 I am confused that how to use the lib of gator1/gophercloud. The master branch is still using the vendor/github.com/gophercloud, and if I run 'govendor fetch github.com/gator1/gophercloud', it adds a new folder of vendor/github.com/gator1/gophercloud. How to sync the codes in gator1/gophercloud into the vendor/github.com/gophercloud? Does copy the codes in gatro1/gophercloud into the vendor/github.com/gophercloud and run command like 'govendor fetch github.com/gophercloud/gophercloud/openstack/cloudeyeservice' to add a config in vendor.json?

**Comment by gator1** [comment]
@zengchen1024 I kind of cheated. so I put gophercloud from gator1 to my local directory
~/go/src/gophercloud. Then use govendor update gophercloud to do it.

**Comment by zengchen1024** [comment]
@gator1 I got it. Thanks!  I have  submitted another PR of resource SMN (https://github.com/gator1/gophercloud/pull/2).  I have to wait that PR to be merged before I update this PR, otherwise the codes of SMN in folder of vendor/github.com/gophercloud will be lost when I run 'govendor update gophercloud'.  I am sorry to block your work!

**Comment by gator1** [comment]
No problem. You are in. Thanks!

**Comment by gator1** [comment]
please integration branch for future pull request
